### DISCUSSION TITLE
Ram consumption

### DISCUSSION THREAD

**Discussion by OmarEzQED** [cypress-io/cypress#24363] [Questions and Help]
Anyone has the issue, that after the new update, cypress has started to take a large amount of ram?

**Comment by krisodb** [comment]
I've seen the same again when with 10.10.0 running through test pipeline on azure. (image with 2G)
Back to cypress 8.7.
On local machine I see memory usage increasing but less an issue there given enough ram.

**Reply by OmarEzQED** [reply]
I have 16 GB of ram and it is taking sometimes above 30%

**Reply by krisodb** [reply]
have a look at: Memory leak in long running single test https://github.com/cypress-io/cypress/issues/4164#top

**Reply by OmarEzQED** [reply]
Thank you very much
